packages/components/src/beta-badge/BetaBadge.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { __ } from "@wordpress/i18n";
+
+/**
+ * Function for the BetaBadge component.
+ *
+ * @param {bool} inLabel Whether the BetaBadge is displayed close to a Label.
+ *
+ * @returns {React.Component} The BetaBadge.
+ */
+const BetaBadge = ( { inLabel } ) => (
+	<span className={ inLabel ? "yoast-badge yoast-badge__in-label yoast-beta-badge" : "yoast-badge yoast-beta-badge" }>
```

Comment:
`yoast-badge__in-label` indicates this is an 'element' through BEM class naming convention, though this is actually a 'modifier' of the badge class and should be named `yoast-badge--in-label`. So the hyphens instead of the underscores are the convention here. Feel free to ignore.

```suggestion
	<span className={ inLabel ? "yoast-badge yoast-badge--in-label yoast-beta-badge" : "yoast-badge yoast-beta-badge" }>
```